src/main/java/com/portatlas/request/RequestMethod.java

```diff
@@ -0,0 +1,20 @@
+package com.portatlas.request;
+
+public enum RequestMethod {
+    GET("GET"),
```

Comment:
I think you can still return strings from an enum. I'm fine with what you did with the static final Strings in those classes though. Maybe you can look into this another time